### PR TITLE
SnackBarMessage

### PR DESCRIPTION
**Pull request by Sujith-Barve**
Two SnackBar Message For Success Submission and Failed Submission of Feedback

### CODE REVIEW

File: locale/English.json [hunk 1]
```diff
@@ -46,8 +46,9 @@
       "btnRisky": "(Dangerous!)",
       "qrContentCopied": "Copied QR Content",
       "translatedText": "Translated Text",
-      "feedbackSuccess":"You have Successfully Submitted Feedback.Thank You",
-      "feedbackFailure": "Something went wrong...Please try again "
+      "feedbackSuccess": "You have Successfully Submitted Feedback.Thank You",
```
Comment (amarts) [inline locale/English.json]: Space after fullstop. and no caps inbetween.

"You have successfully submitted feedback. Thank You"

File: locale/English.json [hunk 2]
```diff
@@ -46,8 +46,9 @@
...
+      "feedbackFailure": "Something went wrong...Please try again ",
```
Comment (amarts) [inline locale/English.json]: "Something went wrong... Please try again."

Btw, I am not a fan of telling "something", that doesn't give any hint, and for users it feels like we are clueless. I would say, "Failed to submit feedback at this time. Please try again in sometime" may sound better. 

@sankarshanmukhopadhyay @mankuthimma what do you think?

File: locale/English.json [hunk 3]
```diff
@@ -46,8 +46,9 @@
...
+      "feedbackEmpty": "Feedback Can not be empty"
```
Comment (amarts) [inline locale/English.json]: again, no caps inbetween:

"Feedback can not be empty."

**Review by amarts — CHANGES_REQUESTED** [review]

File: locale/English.json (hunk 2 above)
Comment (sankarshanmukhopadhyay) [reply locale/English.json]: Perhaps something like "Oops! Your feedback didn't get through ... Please try again!" might keep the tone light and engaging.

**Review by amarts — APPROVED** [review]

**Pull request merged**